user_guide_src/source/incoming/routing/069.php

```diff
@@ -4,7 +4,10 @@
 // Would execute the show404 method of the App\Errors class
 $routes->set404Override('App\Errors::show404');
 
-// Will display a custom view
+// Will display a custom view.
 $routes->set404Override(static function () {
-    echo view('my_errors/not_found.html');
+    // If you want to get the URI segments.
+    $segments = request()->getUri()->getSegments();
```

Comment:
What is the context of using `$segments`? Should we assign it to the `view()`?

Reply:
In normal controllers we can get URI segment as method parameters, but 404 controller cannot.
So I add the sample code.

A use case is that getting a segment showing the locale and display the localized 404 page.

Reply:
Ok, I get it now. Thanks.